README.md

```diff
@@ -0,0 +1,161 @@
+If you, for any reason, estimate that you cannot finish the assignment within the timeframe please be communicative about an ETA.
+
+## How we evaluate
+When we evaluate your home assignemnt, before the onsite workshop, these are the main areas that we prioritize:
+
+#### Does your Git history reveal your workflow?
+When taking a first look at your Git repo we assess how you've iterated on the task.
+We do this by glancing through your Git commits. This will give us an idea of how you've priorised the work and how you've iterated on the development.
+E.g. how are business critical vs. non-critical implementations balanced?
+Therefore make sure that your commit messages provide specific context, e.g.:
+```
+# AVOID
+some fixes
+
+# GOOD
+fix(list): fix issues with the tasks list
+```
+
+#### Does your solution work out-of-the-box?
+Next, we read through your `README.md` and follow your instructions.
+Did you add custom steps? Do they work? Or, does it requires manual debugging to get up and running?
+
+#### To what degree are the requirements met?
+Once everything is up and running, dev server, API etc., we open your solution in a browser and click around.
+Does it work? Are the requirements met? If not, we ask "why" and seek for answers in `README.md` or code-level comments.
+
+#### How does it work?
+Provided that all of the acceptance criteria are up to standard, we start looking into how you implemented the solution.
+I.e. How did you decide to structure the application? How is it architectured? How do you manage state in the application? How scalable is your application?
+How performant is it? Which coding patterns and constructs do you utilise?
+Where and how do you handle edge cases? What about error handling? Is it accessible?
+
+#### How's the sense of quality?
+While assessing the inner workings we also take a step back and look at the overall solution.
+How is it crafted? Does it feel solid, yet simple, in its execution? Is it built using common sense and pragmatic thinking?
+Is it well-tested? Is a good understanding of UX and application patterns applied?
+Is there a coherent idea submerging from the code and structure? What's the level of code smells?
+Are linting rules strictly followed?
+
+Equally there are areas that will **NOT** weight heavily in our evaluation:
+
+#### Visual design quality
+We do not evaluate the overall visual look and feel.
+We expect that your main focus is spend on developing and fine-tuning application logic.
+We do, however, expect that you show an understanding of common UX patterns and best practices, and can make good use of these when making UI/UX decisions.
+
+#### Timeframe
+If nothing else is agreed, we expect the submission to be handed in within 10 days. However, we do not evaluate you on the time you spent on the assignment.
+I.e. we do not count the days from when you received the assignment till you submit, neither do we consider time intervals between your Git commits.
```

Comment:
_until_ instead of _till_.

Reply:
I'm pretty sure `until` and `till` can be used interchangeably. Not that my feelings are strong about this one, just curious why it should changed?